ci-automation/ci-config.env

```diff
@@ -53,3 +53,14 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Azure --
+AZURE_HYPER_V_GENERATION="V1"
+AZURE_IMAGE_NAME="flatcar_production_azure_image.vhd"
+AZURE_LOCATION="westeurope"
+AZURE_MACHINE_SIZE="Standard_D2s_v4"
```

Comment:
Would be good to be able to overwrite the instance type from the job

Reply:
The way I was approaching was to overwrite the `sdk_container/.env` if the Jenkins params are passed. https://github.com/kinvolk/jenkins-os/blob/flatcar-master/container/test.groovy#L63

The ci-config.env would serve as the configuration for basic setup, or running the items locally.

Reply:
That won't work because `ci-automation/test.sh` sources `ci-automation/ci_automation_common.sh` which sources `ci-automation/ci-config.env` and will then overwrite the values.

Reply:
Maybe we should write those env vars like:

```bash
: ${AZURE_HYPER_V_GENERATION:="V1"}
: ${AZURE_IMAGE_NAME:="flatcar_production_azure_image.vhd"}
: ${AZURE_LOCATION:="westeurope"}
: ${AZURE_MACHINE_SIZE:="Standard_D2s_v4"}
```